src/plugins/util/OverlayPlugin.js

```diff
@@ -91,4 +93,12 @@ module.exports = class OverlayPlugin {
             this.restoreState();
         }
     }
+
+    toggleWidget(widget,toolbarBtnImage) {
+        this.instance.store.dispatch({type: 'OVERLAY_OPEN', payload: widget.classList.contains('gm-hidden')});
+
+        // Toggle display
+        widget.classList.toggle('gm-hidden');
+        toolbarBtnImage.classList.toggle('gm-active');
```

Comment:
I'm not sure I understand why we pass the `toolbarBtnImage` as an arg instead of using `this.toolbarBtnImage` ?